package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-file-upload-api-client",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
This update we must make on the release branch.. and not here.  Next time take care...